components/pack/include/PackFile.h

```diff
@@ -43,6 +43,91 @@ struct G3DTextureHeader
 
 static_assert(sizeof(G3DTextureHeader) == 4 * sizeof(uint32_t));
 
+struct SadAudioMetaDataHeader
+{
+	char unknown[27];
+	uint32_t entries;
+};
+
+struct SadAudioBankSampleTable
+{
+	uint16_t numOfEntries;
+	uint16_t numOfEntries2;
+};
+
+struct SadBlockHeader
+{
+	char name[31];
+	uint32_t size;
+};
+
+enum class AudioBankLoop : uint16_t
+{
+	None,
+	Restart,
+	Once,
+	Overlap,
+};
+
+// TODO: Look for channel count (e.g 2)
+// TODO: Look for word length (e.g 16);
+struct AudioBankSample
+{
+	char audioSampleName[255];
+	int32_t unknown0;
+	int32_t id;
+	int32_t isBank;
+	uint32_t size;
+	uint32_t relativeOffset;
+	int32_t clone;
+	int16_t group;
+	int16_t atmosGroup;
+	int32_t unknown4;
+	int32_t unknown5;
+	int16_t unknown6a;
+	int16_t unknown6b;
+	uint32_t sampleRate;
+	int16_t unknownOthera;
+	int16_t unknownOtherb;
+	int16_t unknown7a;
+	int16_t unknown7b;
+	int32_t unknown8;
+	int32_t lStart;
+	int32_t lEnd;
+	char sampleDescription[255];
```

Comment:
this and `audioSampleName` are not aligned on 32 bits. Are we sure it's not a size 256? Either way most compilers will pad an extra char after the string

Reply:
Yep it needs to be 256